### PR TITLE
webapp/partials/*.jsp: change to 2-space indentation #7352

### PR DESCRIPTION
**Pull request by sujeet14108**
Fixes partially #7352

### CODE REVIEW

**Comment by sujeet14108** [comment]
@chowyb Please review :)

**Comment by sujeet14108** [comment]
Ready for review :)

**Comment by wkurniawan07** [comment]
@sujeet14108 there's a new merge conflict that needs to be resolved here

**Comment by sujeet14108** [comment]
@wkurniawan07 Done.

File: src/main/webapp/partials/instructorHelpSessions.jsp [hunk 1]
```diff
@@ -6932,6 +6932,8 @@
           <br>
 
           <br> To setup the question, enter the main question text, and add the options for the students to rank. You can configure if students can give the same rank multiple times.
+                    <br> You can also configure the minimum and (or) maximum options to be ranked. When the minimum options to be ranked restriction is enabled, it will ensure that the respondent ranks at least the number of options mentioned in the restriction.
+                    <br> Similarly, when maximum options to be ranked restriction is enabled, it will ensure that the respondent ranks at most the number of options mentioned in the restriction.
```
Comment (wkurniawan07) [inline src/main/webapp/partials/instructorHelpSessions.jsp]: @sujeet14108 do be careful and review your own changes before asking for review. You should be able to tell what is not right about the above two lines.

**Pull request merged**